File: Morning Snack/Python/lowest_common.py
user_input_one = int(input("Enter a number: "))
user_input_two = int(input("Enter a number: "))

lowest_common = 1
highest_common = 1

for count in range (1, user_input_one and user_input_two):
    if(user_input_one % count == 0 and user_input_two % count == 0):        
        highest_common = count

lowest_common = (user_input_one * user_input_two) // highest_common;
print(lowest_common);

File: Morning Snack/JavaScript/lowestCommon.js
const input = require('prompt-sync')();
let userInputOne = input('Enter number: ');
let userInputTwo = input('Enter number: ');
let lowestcommon;
let count;

for(count = 1; count <= userInputOne && count <= userInputTwo; count++){
    if(userInputOne % count === 0 && userInputTwo % count === 0){
        
                highestcommon = count;

        }
      } 

    lowestCommon = Math.floor((userInputOne * userInputTwo)/highestcommon);
        console.log(lowestCommon);
   


